### PR TITLE
Adding new problem

### PR DESCRIPTION
**Pull request by JChen5543**

### CODE REVIEW

**Comment by LouisLe324** [comment]
Everything looks good, just need to change the category name to list-2

File: pull-primes/description.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+Write a function `primes(list)`, where the list contains 4 positive integers between 0 and 169 (not included), that determines if any of the numbers are primes and prints a new list that includes only the primes (make sure that the primes are in ascending order), if there are none, print `There are no prime numbers in this list`
```
Comment (emaj9) [inline pull-primes/description.md:2]: but `primes` isn't defined in the starter? 
do they need a function that determines if a number is prime? if this solution contains two functiosn (the list filtering one and a func to determine if a number is prime), give both these function headers in the starter
I also find the problem description is alittle unclear...

File: pull-primes/description.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+Write a function `primes(list)`, where the list contains 4 positive integers between 0 and 169 (not included), that determines if any of the numbers are primes and prints a new list that includes only the primes (make sure that the primes are in ascending order), if there are none, print `There are no prime numbers in this list`
+For the sake of simplicity, assume that we only care for the prime factors 2, 3, 5, 7 and 11
```
Comment (emaj9) [inline pull-primes/description.md:3]: what does this mean?

**Comment by emaj9** [comment]
Please address my comments @JChen5543 then resubmit when it's fixed

**Pull request merged**